CHANGELOG.md

```diff
@@ -23,6 +23,8 @@ Guide: https://keepachangelog.com/en/1.0.0/
 - [Core] Remove Swifter library dependency from MapboxSearch target (only used in Test targets)
 - [Core] Change AbstractSearchEngine.init `supportSBS: Bool = false` parameter to `apiType: ApiType = .SBS`. This changes the default API engine for discover/category and other API requests to SBS. Add ApiType enum to represent non-Autofill and non-PlaceAutocomplete SearchEngine API types.
 
+**MapboxCoreSearch**: v2.0.0-alpha.13
```

Comment:
Are there any changes between alpha.9 and alpha.13 that we want to highlight in this public changelog?

Reply:
@aokj4ck just FYI, native changelist can be found [here](https://github.com/mapbox/mapbox-search-sdk-v2/blob/main/CHANGELOG.md?plain=1#L26-L53) .

Reply:
Thank you @globaltrouble; no the changes are relevant to offline search and Android